reel-picker/src/main/java/com/innowisegroup/reelpicker/datetime/LocalTime.kt

```diff
@@ -44,10 +48,6 @@ class LocalTime private constructor(val hour: Int, val minute: Int) : Serializab
                     && isWithinMinMaxRange(time.minute, minTime.minute, maxTime.minute)
         }
 
-        private fun create(hour: Int, minute: Int): LocalTime {
-            require(hour in MIN_HOUR..MAX_HOUR) { "Invalid hours value" }
-            require(minute in MIN_MINUTE..MAX_MINUTE) { "Invalid minutes value" }
-            return LocalTime(hour, minute)
-        }
+        private fun create(hour: Int, minute: Int): LocalTime = LocalTime(hour, minute)
```

Comment:
Seems like we don't need this function anymore, since it calls only from one place now. Remove it